tests/unit/configurations/generalService.test.ts

```diff
@@ -51,6 +52,20 @@ describe('Configuration: General Service', () => {
     expect(res.patientConfiguration).toBeDefined();
   });
 
+  it('Should not transform custom data in the response', async () => {
```

Comment:
I would add in response **of the get request**, since there are also other end points tested here.